core/spec/lib/spree/core/testing_support/factories/taxon_factory_spec.rb

```diff
@@ -13,6 +13,12 @@
     context "when no taxonomy is given" do
       subject { create(:taxon) }
 
+      before do
+        # ensure that the subject is not accidentally
+        # getting valid parent and taxonomy ids
```

Comment:
This comment is clear only in combination with the commit description. So I propose to have still a separate commit that adds this `before` only and explains why we need it, plus, squashing the other part of the commit (the one related to  `core/lib/spree/testing_support/factories/taxon_factory.rb`) in a previous commit, so this one only contains this extra safeguard.

Reply:
I will change the commit title, thanks!